src/sass/base/vd-tweaks.scss

```diff
@@ -84,3 +84,9 @@ a:focus h1 {
 	border: none;
 	font-weight: 300;
 }
+
+.spacer__med {
+	@include AU-media( md ) {
+		margin-bottom: 3rem;
```

Comment:
@sirleech and this bit?

`@include AU-space(margin-bottom, 3unit);`

Reply:
Here is the margin fix:
https://guides.service.gov.au/content-strategy/identify-business-needs/

Added margin bottom:
https://service-manual.apps.y.cld.gov.au/content-strategy/identify-business-needs/